### PR TITLE
📝 add readme

### PR DESCRIPTION
**Pull request by hgwood**

### CODE REVIEW

**Review by Bugsyaya — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+types). Extract complex or common types to TypeScript files with the suffix
+`.types.ts` (see `config.js` and `config.types.ts` for an example). There is no
+need to import those files in other files.
+
+> ℹ Due to limitations of TypeScript, types files cannot be named `example.ts`
+> or `example.d.ts` if `example.js` also exists.
+
+## Formatting
+
+This projects uses prettier. Check formatting with `npm run prettier:check` or
```
Comment (pyaillet) [inline README.md:131]:
```suggestion
This project uses prettier. Check formatting with `npm run prettier:check` or
```

**Review by pyaillet — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+The value for `key` specifies the bearer token that the client must call with.
+Values can be interpolated into `path`. These values are expected to be
+specified using the query parameters of the incoming request. For this example,
+calling `/example_path?username=example@example.com` would result in Proxybeez
+calling `/query/users?filter=username==example@example.com&fields=uuid,username`
+on Alibeez. Interpolated values are escaped for usage in query parameters.
+
+## Processors
+
+On either tenants on paths, the configuration can define a list of processors to
```
Comment (EmrysMyrddin) [inline README.md:72]: I supposed their is a mistake here ?
```suggestion
On either tenants or paths, the configuration can define a list of processors to
```

**Review by EmrysMyrddin — COMMENTED** [review]
I'm not sure to understand how tokens are configured. You have to add a token to each path you add in the configuration ?

**Comment by hgwood** [comment]
> I'm not sure to understand how tokens are configured. You have to add a token to each path you add in the configuration ?

Yes. Though we can use the same token for multiple paths if we wish.

**Review by EmrysMyrddin — APPROVED** [review]

**Pull request merged**